utils/rpms/daos.spec

```diff
@@ -217,6 +217,7 @@ mkdir -p %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{server_svc_name} %{?buildroot}/%{_unitdir}
 install -m 644 utils/systemd/%{agent_svc_name} %{?buildroot}/%{_unitdir}
 mkdir -p %{?buildroot}/%{conf_dir}/certs/clients
+mv %{?buildroot}/%{_prefix}/etc/bash_completion.d %{?buildroot}/%{_sysconfdir}
```

Comment:
How come this isn't covered by the patch you did a while ago to move `/usr/etc/` to `/etc/`?

Reply:
```%define conf_dir %{_sysconfdir}/daos

scons %{?_smp_mflags}      \
      --config=force       \
      --no-rpath           \
      USE_INSTALLED=all    \
      CONF_DIR=%{conf_dir} \
      PREFIX=%{?buildroot}
```

Seemed like overkill to add a new scons variable just for this...

Reply:
Oh yes.  I recall now.  We are telling `scons` to use `/etc/daos`, not `daos/' in `/etc/`.